mitodl/ocw-hugo-themes
# fix: expand description button style

### What are the relevant tickets?
Related to https://github.com/mitodl/ocw-hugo-themes/pull/1371 and https://github.com/mitodl/hq/issues/3858

### Description (What does it do?)

In https://github.com/mitodl/ocw-hugo-themes/pull/1371 we fixed a markdown rendering issue. The fix changes the HTML layout of the `collapsed-description` div.

We used to have this layout
```html
<div>
  text text text <button></button>
</div>
```

The fix changes it to 
```html
<div>
  <p>text text text</p>
  <button></button>
</div>
```

This introduces a line break that changes how the "See more" button is placed.

<img width="779" alt="Screenshot 2024-03-28 at 12 00 22 PM" src="https://github.com/mitodl/ocw-hugo-themes/assets/71316217/05978a8f-3766-4baf-aedf-8182fa9ce804">

> What it should look like


<img width="808" alt="Screenshot 2024-03-28 at 12 00 41 PM" src="https://github.com/mitodl/ocw-hugo-themes/assets/71316217/8efa6582-3888-41d6-9d4b-41f0ff733b88">

> What it looks like on RC right now


### Screenshots (if appropriate):


<img width="767" alt="Screenshot 2024-03-28 at 12 04 25 PM" src="https://github.com/mitodl/ocw-hugo-themes/assets/71316217/860fa03d-4d8a-4134-bc21-d638471b3950">

### How can this be tested?

1. Grab contents of [this course](https://github.mit.edu/ocw-content-rc/sts.050-spring-2016).
2. Navigate to your local `ocw-hugo-themes` setup.
3. Run
    ```
    yarn start course sts.050-spring-2016
    ```
4. Open http://localhost:3000/
5. Observe the homepage.
6. Expect to see the description (collapses and expanded) rendered correctly. Please see the screenshot for reference.
